crates/re_renderer/src/context.rs

```diff
@@ -399,6 +394,40 @@ impl RenderContext {
                 .push(self.queue.submit([command_buffer]));
         }
     }
+
+    /// Gets a renderer with the specified type, initializing it if necessary.
+    pub fn get_renderer<R: 'static + Renderer + Send + Sync>(
```

Comment:
`get_renderer` breaks the standard API naming guidelines: https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter